Bottom: 1f00f6ee1996539a85f6750c74d43b53408b42fa
Top:    be78ff4086c7137b009e5d99b2f4b212d6bd73cd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 13:25:25 +0000

Refresh of attempt-at-producing-inter

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index e183993..b187b00 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -533,7 +533,6 @@ object3d_from_contours (POLYAREA *contours,
       DDATA (edges[1 * npoints + i]) = vertices[1 * npoints + next_i_around_ct];
       ODATA (edges[2 * npoints + i]) = vertices[0 * npoints + i];
       DDATA (edges[2 * npoints + i]) = vertices[1 * npoints + i];
-      /* XXX: Do we need to differently with this for inverted contours? */
 
       if (get_contour_edge_n_is_round (ct, offset_in_ct)) {
         double cx;
